packages/ice/src/ice.ts

```diff
@@ -921,6 +928,8 @@ export class Connection {
     for (const protocol of this.protocols) {
       if (
         protocol.localCandidate?.canPairWith(remoteCandidate) &&
+        (!this.options.forceTurn || protocol.type === "turn") &&
```

Comment:
nit: I think `!(this.options.forceTurn && protocol.type !== "turn")` might be a little more readable here?

Reply:
tbh I find conditionals less readable with a not around the whole thing, but if you feel strongly about it I'll change it

Reply:
Haha as it is I had to look at it for a minute to figure out what was implied. I find "if it's *not* the case that (we're forcing TURN *and* the candidate is not a relay candidate)" easier to parse, but I guess it's just personal preference.